Sources/DataConnect.swift

```diff
@@ -89,14 +92,21 @@ public class DataConnect {
         callerSDKType: callerSDKType
       )
 
-      operationsManager = OperationsManager(grpcClient: grpcClient)
+      if let cache {
+        self.cache = Cache(config: cache.config, dataConnect: self)
```

Comment:
Won't there be a strong retain cycle between `Cache` and `DataConnect`?

Reply:
Yes, thanks - good catch. This would result in a retain cycle. 

Fixed to make this a weak reference.